Formula/llvm.rb

```diff
@@ -113,10 +113,18 @@ def install
       args << "-DLLVM_BUILD_LLVM_C_DYLIB=ON"
       args << "-DLLVM_ENABLE_LIBCXX=ON"
       args << "-DLLVM_CREATE_XCODE_TOOLCHAIN=#{MacOS::Xcode.installed? ? "ON" : "OFF"}"
-      args << "-DRUNTIMES_CMAKE_ARGS=-DCMAKE_INSTALL_RPATH=@loader_path/../lib"
+      args << "-DRUNTIMES_CMAKE_ARGS=-DCMAKE_INSTALL_RPATH=#{rpath}"
 
       sdk = MacOS.sdk_path_if_needed
       args << "-DDEFAULT_SYSROOT=#{sdk}" if sdk
+
+      extra_includes = [HOMEBREW_PREFIX/"include"]
```

Comment:
I'm hoping this won't affect the bottle being `cellar: :any`, since this is stored in a text file. If it makes the bottle non-relocatable we can drop it.

Reply:
`C_INCLUDE_DIRS` should be baked in to clang binaries so it will affect relocatability.

Reply:
`C_INCLUDE_DIRS` should be baked in to clang binaries so it will affect relocatability.

Reply:
`DEFAULT_SYSROOT` isn't (at least, not in a way you can find with `strings`), so I was maybe a bit hopeful here.